### DISCUSSION TITLE
load data based on scroll state

### DISCUSSION THREAD

**Discussion by virtualpathum** [patrykandpatrick/vico#676] [Questions] [answered]
Hi, I need to show only a limited set of data due to a space constraint. When the user scrolls, I want to automatically load the rest of the data (similar to pagination in lists.)

I need to increase the font size of the bottom Axis for better readability. So, I want to show, eg 8 records at first and when the user scrolls, the chart should loads the remaining data with increased font size. 

the current behaviour is when I load all data, instead of showing the bottom line labels, the chart is showing some dots.

is it possible to do this or any other better approach to show the chart more readable way?

**Comment by Gowsky** [comment, marked as answer]
Hi!
>instead of showing the bottom line labels, the chart is showing some dots.

I don't know what the font size has to do with it, but it sounds like the zoom level is too low, and the labels don't fit. Or the scrolling is disabled. You can adjust the initial, minimum, and maximum zoom levels, as well as scrolling as described [here](https://patrykandpatrick.com/vico/wiki/cartesian-charts/hosts#scroll-and-zoom).

If you meant something else, could you attach an example image of what you have and point out the exact problem?

**Reply by virtualpathum** [reply]
@Gowsky Thanks for the reply. I was able to resolve the issue.. so far so good..

**Reply by Gowsky** [reply]
You're welcome!
